frontend/src/components/Forms/PolicyForm/index.module.scss

```diff
@@ -0,0 +1,19 @@
+.headingWrapper {
+  margin-bottom: 104px;
+  text-align: center;
+}
+
+.checkboxWrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding: 0 20px;
+}
+
+.termsAndConditions {
+  margin-bottom: 32px;
+
+  a {
```

Comment:
Does the link have padding by default? 🤔

Reply:
If we answer is yes, shouldn't we move this to `normalize`?

Reply:
It didn't, was just a style I forgot to erase while testing things out - it has been removed already